xblock/core.py

```diff
@@ -221,7 +238,7 @@ def add_xml_to_node(self, node):
 
 class XBlockAside(XmlSerializationMixin, ScopedStorageMixin, RuntimeServicesMixin, HandlersMixin, SharedBlockBase):
     """
-    This mixin allows Xblock-like class to declare that it provides aside functionality.
+    This class allows XBlock-like classes to declare that they provides aside functionality.
```

Comment:
If the presence of this Mixin in the class hierarchy indicates Aside support, then the `supports_asides` method is superfluous.